Code/Framework/AzGameFramework/AzGameFramework/Application/GameApplication.cpp

```diff
@@ -45,6 +45,8 @@ namespace AzGameFramework
             enginePakPath = AZ::IO::FixedMaxPath(AZ::Utils::GetExecutableDirectory()) / "engine.pak";
             m_archive->OpenPack("@products@", enginePakPath.Native());
         }
+        constexpr AZStd::string_view paksFolder = "@products@/*.pak"; // (@products@ assumed)
+        m_archive->OpenPacks(paksFolder);
```

Comment:
I think we want to use OpenPacks because we don't want to keep to the pak file extension forever, do we? I thought building a replacement for paks was on the list of work to be done?